apps/viewer/src/hooks/usePlaybackControl.ts

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+
+import { ViewerVideoViewProps } from '../components/viewer-video-view';
+
+const usePlaybackControl = (sourceIds: string[]) => {
+  const [playbackControl, setPlaybackControl] = useState<Record<string, ViewerVideoViewProps['controls']>>({});
```

Comment:
This is not easy to understand Record<string, ViewerVideoViewProps['controls'].  What's the meaning of string type here? SourceID?
So this hook coupled with ViewerVideoView's properties here, not a good design actually. This hook only needs to know what settings or options to maintain as it's states.
Could we move this into ViewerVideoView?